sctk/Cargo.toml

```diff
@@ -14,12 +14,14 @@ multi_window = []
 [dependencies]
 log = "0.4"
 thiserror = "1.0"
-sctk = { package = "smithay-client-toolkit", git = "https://github.com/Smithay/client-toolkit", rev = "69bffe5", features = ["calloop"] }
-raw-window-handle = "0.5.0"
-enum-repr = "0.2.6"
+sctk = { package = "smithay-client-toolkit", git = "https://github.com/smithay/client-toolkit" }
```

Comment:
We've been pinning a specific `rev` of sctk/smithay here and elsewhere to make sure this keeps building and doesn't break due to a change in the repository. Which is a bit annoying, but we probably still want to do that.